data/monster/monsters/starving_wolf.xml

```diff
@@ -36,7 +38,7 @@
 		<voice sentence="Yoooohhuuuu!" />
 	</voices>
 	<loot>
-		<item id="220" chance="64730" /><!-- dirty fur -->
```

Comment:
@DSpeichert @EvilHero90  do you have any idea on why this didn't popped an error btw? item id 220 doesn't exist (it was a typo), but server didn't noticed this even with forceMonsterTypesOnLoad enabled

Reply:
https://github.com/otland/forgottenserver/blob/4fcb51fb86d6e77150e2279d56c66bcb1c48cb36/src/monsters.cpp#L1354

There is no check if an item with that id exists compared to one with a string / name.

https://github.com/otland/forgottenserver/blob/4fcb51fb86d6e77150e2279d56c66bcb1c48cb36/src/monsters.cpp#L1359-L1362

Reply:
oh makes sense, hope someone could add it

Reply:
@EPuncker you can add the warning as follows:
```c++
if ((attr = node.attribute("id"))) {
		int32_t id = pugi::cast<int32_t>(attr.value());
		const ItemType& it = Item::items.getItemType(id);
		if (it.name.empty()) {
			std::cout << "[Warning - Monsters::loadMonster] Unknown loot item id \"" << id << "\". " << std::endl;
			return false;
		}

		lootBlock.id = id;

	} else if ((attr = node.attribute("name"))) {
```
I already tried it and it seems to work perfectly.